samples/drivers/mbox/src/main.c

```diff
@@ -19,24 +18,25 @@ static void callback(const struct device *dev, uint32_t channel,
 
 int main(void)
 {
-	const struct device *dev;
+	int ret;
+	const struct device *const dev = DEVICE_DT_GET(DT_NODELABEL(mbox));
 
 	printk("Hello from APP\n");
```

Comment:
You could consider renaming this to something like MAIN or HOST. This CPU does not have to do anything with an application.

Reply:
Sample needs a bit of love, I’ll rework it as part of the https://github.com/zephyrproject-rtos/zephyr/pull/69390 when I rebase it